src/main/java/com/jcraft/jsch/DHECN.java

```diff
@@ -109,6 +108,7 @@ public boolean next(Buffer _buf) throws Exception{
     int i,j;
     switch(state){
     case SSH_MSG_KEX_ECDH_REPLY:
+                JSch.getLogger().log( Logger.INFO, "Received SSH_MSG_KEX_ECDH_REPLY" );
```

Comment:
I think this should be conditioned on `if(JSch.getLogger().isEnabled(Logger.INFO)){`.